include/net/net_pkt.h

```diff
@@ -2319,9 +2319,9 @@ struct net_pkt *net_pkt_clone(struct net_pkt *pkt, s32_t timeout);
 /**
  * @brief Read some data from a net_pkt
  *
- * Note: net_pkt's cursor should be properly initialized and,
- *       eventually, properly positioned using net_pkt_skip.
- *       Cursor will be updated according to parameters.
+ * Note: net_pkt's cursor should be properly initialized and, if needed,
+ *       positioned using net_pkt_skip.
+ *       Cursor will be updated after the operation.
```

Comment:
Let's be consistent, reuse the exact same sentence as above (line 2297, with comments)

Reply:
I'll update everything consistently, there're more cases there.

Reply:
this sentence differs still.

Should "position" be added?